smallrye-reactive-messaging-aws-sns/pom.xml

```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.smallrye.reactive</groupId>
+    <artifactId>smallrye-reactive-messaging</artifactId>
+    <version>1.0.4-SNAPSHOT</version>
+  </parent>
+  <properties>
+  	<vertx.version>3.8.0</vertx.version>
+  </properties>
+  <artifactId>smallrye-reactive-messaging-aws-sns</artifactId>
+  <name>Smallrye Reactive Messaging for AWS SNS</name>
+  <description>A module for Smallrye reactive messaging integration with AWS Simple Notification Service (SNS).</description>
+    <dependencyManagement>
+    <dependencies>
+      <dependency>
+	      <groupId>software.amazon.awssdk</groupId>
+	      <artifactId>bom</artifactId>
+	      <version>2.7.13</version>
```

Comment:
Heard today they released a new version (2.8.0) that is compatible with the netty version we are using.

Reply:
Will upgrade and test.